styleguide/source/_patterns/02-organisms/03-sections/02-card-row-ec/02-card-row-ec-four-up.twig

```diff
@@ -1,6 +1,30 @@
-<div class="layout layout-four-up">
-    <div class="layout_item">{% include 'molecules-card-image-landscape' %}</div>
-    <div class="layout_item">{% include 'molecules-card-image-landscape' %}</div>
-    <div class="layout_item">{% include 'molecules-card-image-landscape' %}</div>
-    <div class="layout_item">{% include 'molecules-card-image-landscape' %}</div>
-</div>
\ No newline at end of file
+{% if cardType == 'landscape' %}
```

Comment:
I like that this is more succinct, but it might not be immediately obvious that landscape cards are the default here. This is probably fine and someone familiar with Twig syntax should be be able to figure out what's going on with some poking around.

Wanted to point that out; despite this, I've made these changes [here](https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/157/commits/42856000e883b34e26e6d70712e55897e751ef76).